### DISCUSSION TITLE
4.6.3

### DISCUSSION THREAD

**Discussion by jrosdahl** [ccache/ccache#1151] [General]
* [Generic source code release (tar.gz)](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3.tar.gz) ([signature](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3.tar.gz.asc))
* [Generic source code release (tar.xz)](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3.tar.xz.asc))
* [Linux x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-linux-x86_64.tar.xz) ([signature](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-linux-x86_64.tar.xz.asc))
* [Windows i686 binary release](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-windows-i686.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-windows-i686.zip.asc))
* [Windows x86_64 binary release](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-windows-x86_64.zip) ([signature](https://github.com/ccache/ccache/releases/download/v4.6.3/ccache-4.6.3-windows-x86_64.zip.asc))
* [Release notes](https://ccache.dev/releasenotes.html#_ccache_4_6_3)
* [Manual](https://ccache.dev/manual/4.6.3.html)

<hr /><em>This discussion was created from the release <a href='https://github.com/ccache/ccache/releases/tag/v4.6.3'>4.6.3</a>.</em>

**Comment by silverqx** [comment]
The msvc support is still better and better, I'm using the ccache in GitHub actions with the msvc and never had any problem. I'm still hoping that the day when it will support also the msvc PCH will come soon and we will use the ccache during development as well 😁 Thx @jrosdahl
